BunqSdk/Http/Pagination.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System.Collections.Generic;
+using Bunq.Sdk.Exception;
+
+namespace Bunq.Sdk.Http
+{
+    public class Pagination
+    {
+        /// <summary>
+        /// Error constants.
+        /// </summary>
+        private const string ERROR_NO_PREVIOUS_PAGE =
+            "Could not generate previous page URL params: there is no previous page.";
+
+        /// <summary>
+        /// URL Param constants.
```

Comment:
Ok I triple checked this time haha,  not generated right 😝. Below you have param without capital `P` so I guess this one should also be lower case ?

Reply:
Ohh OCD! :D I'll fix this :)

Reply:
Done